appserver/ejb/ejb-container/src/main/java/com/sun/ejb/EjbInvocation.java

```diff
@@ -75,32 +84,14 @@
  * EJB(Local)Object/EJB(Local)Home before and after an invocation.
  */
 
-public class EjbInvocation
-    extends ComponentInvocation
-    implements InvocationContext, TransactionOperationsManager,
-         org.glassfish.ejb.api.EJBInvocation, InterceptorManager.AroundInvokeContext
-{
-
+public class EjbInvocation //
+    extends ComponentInvocation //
+    implements InvocationContext, TransactionOperationsManager, EJBInvocation, AroundInvokeContext {
 
     public ComponentContext context;
 
     private TransactionOperationsManager transactionOperationsManager;
 
-    EjbInvocation(String compEnvId, Container container) {
```

Comment:
Constructor was moved behind fields. Too many fields. This class calls for refactoring and splitting.